### PR TITLE
fix: use signed base images

### PR DESCRIPTION
**Pull request by xopham**
works. Only downside is that signed version of docker official images seems somewhat dated. Not clear why that is.

WARN: NOT CLEAR WHETHER DOCKER STILL SUPPORTS THE OFFICIAL IMAGES

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,8 +8,15 @@ COSIGN_VERSION = 1.0.0
 all: docker install
 
 docker:
-	docker build --build-arg COSIGN_VERSION=$(COSIGN_VERSION) -f docker/Dockerfile -t $(IMAGE) .
-	docker build -f docker/Dockerfile.hook -t $(HELM_HOOK_IMAGE) .
+	docker build \
+	  	--disable-content-trust=false \
+		--build-arg COSIGN_VERSION=$(COSIGN_VERSION) \
+		-f docker/Dockerfile \
+		-t $(IMAGE) .
```
[the remark this answers is not in the document]
Comment (xopham) [reply Makefile:15]: hm... :thinking: would keep it I think. The last line is the common docker build :shrug: 

Have you seen my note on the PR? Would really like to get your thoughts on that.

[the remark this answers is not in the document]
Comment (Starkteetje) [reply Makefile:15]: Haven't seen that. In that case, I would not update to signed images I guess. Bc then it's a systemic issue and if fixes are necessary, I would want them over a signature tbf

[the remark this answers is not in the document]
Comment (xopham) [reply Makefile:15]: exactly, I have inquired Docker Inc over the issue

**Comment by xopham** [comment]
not gonna happen

**Pull request closed without merging**